inference-engine/tests/unit/inference_engine/ie_compilation_context_test.cpp

```diff
@@ -349,6 +349,32 @@ TEST(NetworkContext_CNNNetwork, HashWithDifferentMeanValues) {
               NetworkCompilationContext::computeHash(net3, {}));
 }
 
+TEST(NetworkContext_CNNNetwork, HashOfSameMultiThreading) {
```

Comment:
What is the purpose of this test? If I understand right the `computeHash` method doesn't depend on thread context.

Reply:
This test helped me to find out some thread-safety problems inside ngraph serialization, so I'd prefer to keep it to ensure that all internal calculations inside 'computeHash' are thread-safe